external/fv3viz/fv3viz/_plot_cube.py

```diff
@@ -159,16 +160,26 @@ def plot_cube(
         cmap=kwargs.get("cmap"),
         robust=cmap_percentiles_lim,
     )
-
-    _plot_func_short = partial(
-        _plot_cube_axes,
-        lat=mappable_ds.lat.values,
-        lon=mappable_ds.lon.values,
-        latb=mappable_ds.latb.values,
-        lonb=mappable_ds.lonb.values,
-        plotting_function=plotting_function,
-        **kwargs,
-    )
+    if gsrm_name == "fv3":
+        _plot_func_short = partial(
+            _plot_cube_axes,
+            lat=mappable_ds.lat.values,
+            lon=mappable_ds.lon.values,
+            latb=mappable_ds.latb.values,
+            lonb=mappable_ds.lonb.values,
+            plotting_function=plotting_function,
+            **kwargs,
+        )
+    elif gsrm_name == "scream":
+        _plot_func_short = partial(
+            _plot_scream_axes,
+            lat=mappable_ds.lat.values,
+            lon=mappable_ds.lon.values,
+            plotting_function=plotting_function,
```

Comment:
It seems like there is a 1:1 mapping between ["pcolormesh", "contour", and "contourf"] and ["tripcolor", "tricontour", "tricontourf"]. In other words, each of these three plotting functions is the same, just with a cartesian vs unstructured grid. Rather than adding the three new "tri" functions to the acceptable list of `plotting_function` values for the top-level `plot_cube` arg, could you instead for the scream case determine which "tri" function to supply to `_plot_scream_axes` based on whether `plotting_function` is "pcolormesh", "contour", or "contourf"? That way there'd be no need to change how `plot_cube` is called with this argument.

Reply:
Good idea, done!